prow/config/jobs/metal3-io/ironic-ipa-downloader.yaml

```diff
@@ -33,6 +33,10 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: false
+  - name: metal3-centos-e2e-integration-test-release-1-9
```

Comment:
shall we also remove the 1-6 triggers?

Reply:
Removal in another PR.

Reply:
We have agreed to follow best practice of not mixing adding and removing.